core/install.js

```diff
@@ -12,11 +12,25 @@
 var npm = require('npm'),
     deasync = require('deasync'),
     fs = require('fs'),
-    load = deasync(npm.load);
+    load = deasync(npm.load),
+    csHasLoaded = false;
 
 load({loglevel: 'silent'});
 var inst = deasync(npm.commands.install),
-upd = deasync(npm.commands.update),
+    upd = deasync(npm.commands.update),
+
+// inject require modifications into all coffeescript code because babel wont
+coffeescriptRequireInjector = function () {
+    if (!csHasLoaded && global.coffeescriptLoaded) {
+        var cs = require("coffee-script"),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2737018399/issues/source?bid=3282987&fileBranchId=3347522#l25)